### PR TITLE
Fix null scalar formatting

### PR DESCRIPTION
**Pull request by emmatown**

### CODE REVIEW

File: src/formatter.test.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ Markdoc is a **Markdown**-based \`syntax\` and _toolchain_ for creating ~~custom
 [Link](/href   "title")
     ![Alt](/image   "title")
 
-{% callout #id   .class  .class2   a="check" b={"e":{"with space": 5}} c=8 d=[1,    "2",true] %}
+{% callout #id   .class  .class2   a="check" b={"e":{"with space": 5}} c=8 d=[1,    "2",true,  null] %}
```
Comment (mfix-stripe) [inline src/formatter.test.ts:24]: PTAL @rpaul-stripe — I didn't think we supported "null" in the syntax.

Comment (rpaul-stripe) [reply src/formatter.test.ts:24]: We actually do support `null` as a valid scalar in Markdoc tags: https://github.com/markdoc/markdoc/blob/main/src/grammar/tag.pegjs#L119-L120

Comment (mfix-stripe) [reply src/formatter.test.ts:24]: Ooh! Missed that in the grammar. Let's merge this then huh @rpaul-stripe ?

**Review by rpaul-stripe — APPROVED** [review]

**Comment by mfix-stripe** [comment]
Thanks @emmatown!

**Pull request merged**